js/signin.js

```diff
@@ -0,0 +1,114 @@
+const QUERY_SELECTOR = function (select) {
+  return document.querySelector(select);
+};
+
+const $emailInput = QUERY_SELECTOR('#emailInput');
+const $emptyEmailError = QUERY_SELECTOR('#emptyEmailError');
+const $wrongEmailError = QUERY_SELECTOR('#wrongEmailError');
+
+$emailInput.addEventListener('focusout', function (e) {
+  const emailRegEx = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-Za-z0-9\-]+/;
+
+  defaultStyle();
+
+  if (this.value === '') {
+    styleSeting('email', 'empty');
+  } else if (!emailRegEx.test(this.value)) {
+    styleSeting('email', 'wrong');
+  }
+});
+
+const $passwordInput = QUERY_SELECTOR('#passwordInput');
+const $emptyPasswordError = QUERY_SELECTOR('#emptyPasswordError');
+
+$passwordInput.addEventListener('focusout', function (e) {
+  defaultStyle();
+
+  if (this.value === '') {
+    styleSeting('password');
+  }
+});
+
+const $loginButton = QUERY_SELECTOR('#btnLogin');
+const $checkEmailError = QUERY_SELECTOR('#checkEmailError');
+const $checkPasswordError = QUERY_SELECTOR('#checkPasswordError');
+
+function loginBtnClick() {
+  defaultStyle();
+
+  if (
+    $emailInput.value === 'test@codeit.com' &&
+    $passwordInput.value === 'codeit101'
+  ) {
+    $loginButton.href = './folder.html';
+  } else {
+    styleSeting('check');
+  }
```

Comment:
예외 케이스를 먼저 처리하고, 함수의 메인이 되는 기능을 실행시키는 흐름이 더 괜찮을 것 같아요~
추가적으로 자바스크립트를 이용해서 바로 folder 페이지로 넘어가면 좋을 것 같아요!

```suggestion
  if (
    $emailInput.value !== 'test@codeit.com' ||
    $passwordInput.value !== 'codeit101'
  ) {
  styleSeting('check');
  return;
  }
  
window.location.href = './folder.html';
```